plutus-playground-lib/src/Playground/API.hs

```diff
@@ -105,6 +105,7 @@ data EvaluationResult = EvaluationResult
   , resultGraph       :: FlowGraph
   , emulatorLog       :: [EmulatorEvent]
   , fundsDistribution :: [SimulatorWallet]
+  , walletKeys        :: [(PubKey,Wallet)]
```

Comment:
didn't the `Wallet` type used to contain `PubKey`?

Reply:
ah, what about using `SimulatorWallet` instead?

Reply:
Ah yes, that's much nicer. Doing it. 👍